packages/cli/src/commands/generate.js

```diff
@@ -66,9 +71,13 @@ export default {
 
     const generator = await cmd.getGenerator(nuxt)
 
-    await generator.generate({
+    const errors = await generator.generate({
```

Comment:
@loomchild Last request! :see_no_evil:  Can we rename this variable from `errors` to `generateResult` or something similar?

Reply:
Sure, you are right.

Reply:
Done.